eva/documentation_for_adding_new_cmd.md

```diff
@@ -0,0 +1,170 @@
+# Extending a new DDL command to EVA
+This document is the note of tracing implementation of CREATE command. We also list all the file that need to be modified or added to extend a new DDL command to EVA. We hope this tutorial will be helpful when future users trying to implement new command in EVA.
```

Comment:
Can we move this file to docs? Instead of keeping it in the base folder of the repo.

Reply:
Done